templates/faq.html

```diff
@@ -0,0 +1,18 @@
+{% extends 'base.html' %}
```

Comment:
@JimLynx 
"base.html" double-quotes to keep it consistent with the original files.